test/api/contenttype.spec.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+/* eslint-disable promise/always-return */
+import { ContentType } from '../../src/lib/content-type';
+import { stackInstance } from '../utils/stack-instance';
+import { TContentType, TEntry } from './types';
+
+const stack = stackInstance();
+describe('ContentType API test cases', () => {
+  it('should give Entry instance when entry method is called with entryUid', async () => {
+    const result = await makeContentType('header').Entry('blt657fe7db362fea22').fetch<TEntry>();
```

Comment:
remove the credentials from the file @nadeem-cs

Reply:
Resolved in [this](https://github.com/contentstack/contentstack-javascript/pull/110) PR.